rskj-core/src/test/java/org/ethereum/vm/VMTest.java

```diff
@@ -222,6 +223,119 @@ public void testSTATICCALLWithStatusOne() {
         assertTrue(program.getStack().isEmpty());
     }
 
+    // This test should throw an exception because we are reading from the RETURNDATABUFFER
+    // in a non-existent position. This results in an error according to EIP 211
+    @Test(expected = RuntimeException.class)
+    public void doCall_returnDataBuffer_DataIsZeroAfterCallToNonExistentContract() {
```

Comment:
Any test to exercise the behavior BEFORE this activation?

Reply:
added test cases! https://github.com/rsksmart/rskj/pull/1296/commits/db6495bb733eb4b7422db28c7d7eec99aa73c949